builtin/staker/validations.go

```diff
@@ -233,34 +222,22 @@ func (v *validations) ActivateNext(
 	return &id, nil
 }
 
-func (v *validations) UpdateAutoRenew(endorsor thor.Address, id thor.Bytes32, autoRenew bool) error {
+func (v *validations) SignalExit(endorsor thor.Address, id thor.Bytes32) error {
 	validator, err := v.storage.GetValidation(id)
 	if err != nil {
 		return err
 	}
 	if validator.Endorsor != endorsor {
 		return errors.New("invalid endorsor for node")
 	}
-	if validator.AutoRenew == autoRenew {
-		return fmt.Errorf("auto-renewal is already set to %t", autoRenew)
-	}
-
-	validator.AutoRenew = autoRenew
 
 	if validator.Status == StatusActive {
```

Comment:
Maybe:

```suggestion
        if validator.Status != StatusActive {
             return errors.New("can't signal exit while not active")
        }
```

Validators in the queue can withdraw,